### PR TITLE
Install sidekiq for background job processing

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?

So that we can use a queue for background job processing.

## Was the documentation (README.md, openapi.yml) updated?
n/a

### CODE REVIEW

File: config/deploy.rb [hunk 1]
```diff
@@ -25,6 +25,11 @@
 # Default value for linked_dirs is []
 append :linked_dirs, 'log', 'config/settings', 'vendor/bundle'
 
+# Sidekiq configuration (run three processes)
+# see sidekiq.yml for concurrency and queue settings
+set :sidekiq_processes, 3
+
+
```
Comment (mjgiarlo) [inline config/deploy.rb]: Extra blank space. :keyboard:

**Comment by jcoyne** [comment]
@mjgiarlo to what purpose?

**Comment by mjgiarlo** [comment]
@jcoyne :speech_balloon: 
> @mjgiarlo to what purpose?

To spin up workers when running the app in containerland. I'm not sure it's needed, but we do seem to do this in some other projects, *e.g.*, https://github.com/sul-dlss/dor-services-app/blob/master/docker-compose.yml#L35-L52

**Comment by jcoyne** [comment]
@mjgiarlo If it's in containerland, I'm assuming you are using development mode, so it'll be using async as the adapter, right?

**Comment by mjgiarlo** [comment]
> @mjgiarlo If it's in containerland, I'm assuming you are using development mode, so it'll be using async as the adapter, right?

I see we're only using Sidekiq in the production env now, which is A-OK. No problem.

**Pull request merged**